MissingNYC2020USOpen.txt

```diff
@@ -0,0 +1 @@
+Missing being in NYC for the 2020 US Open
```

Comment:
uh no, no missing games :-)

what happens if I hit the 'start a review' button for this file?